docs/hopsworksai/azure/aks_acr_integration.md

```diff
@@ -0,0 +1,229 @@
+# Integration with Azure AKS and ACR
+
+This guide shows how to create a cluster in hopsworks.ai with integrated support for Azure Kubernetes Service (AKS) and Azure Container Registry (ACR). This enables Hopsworks to launch Python jobs, Jupyter servers, and ML model servings on top of AKS.
+
+Hopsworks AKS and ACR integration have four requirements:
+
+1. A virutal network with access to AKS pods and the AKS API servers 
+2. One Azure container registry configured in your account
+3. One AKS cluster
+4. Permissions to the ACR and AKS attached to a user-managed identity
+
+This guide on example setup with a private AKS cluster and public ACR.
+!!! note
+A public AKS cluster means the kubernetes API server is accessible outside the virtual network it is deployed in. Similarly, a public ACR is accessible through the internet.
+
+
+## User-managed identity
+
+!!! note
+The managed identity is attached to the virtual machines that runs inside your subscription. Hence, the permissions only applies to services that runs within your subscription. 
+
+AKS and ACR integration requires some additional permissions to be attached to the managed identity used by the hopsworks cluster. 
+
+To setup the managed identity, go to managed identities in the azure portal and then click add. Place the managed indentity in the same resource group as your are planning to create the hopsworks cluster in. In addition, choose a name for the managed identity, then review and create.
+
+Next go to the created managed identity in order to create role assignments. Go to azure role assignments and add the following roles with scope of the resource group you are planning to use:
```

Comment:
This is adding a managed identity role at the subscription level, which is fine. But we should also have text to describe how to add the role assignment to a resource group.

###Add role assignment at subscription level
.....
###Add role assignment at resource group level
Go to the resource group where you created your managed identity. Select the Access Control (IAM) menu item, then click on "Add", then "Add role Assignment". Select the roles "AcrPull", "AcrPush", "Azure Kubernetes Service Contributor Role" and then select the managed identity that you are assignment them to.

Reply:
I think you got this one the other way around. In the example the roles are applied at resource group level. The scope is chosen when you attach the role. Anyhow, I'll clarify this in the text.

Reply:
Reformulated it a bit in the latest commit.